web/dev.ts

```diff
@@ -109,6 +111,22 @@ export async function setupDevDB() {
         console.log("Inserted Dev Match " + i);
     }
 
+    let pc = new ParticipationCertificate();
+    //pc.uuid = "000000001-0000-0000-0701-1b4c4c526384";
+    pc.uuid = randomBytes(5).toString('hex').toUpperCase();
+    pc.pupil = pupils[0];
+    pc.student = students[0];
+    pc.subjects = JSON.stringify(["Englisch", "Deutsch"]);
```

Comment:
This does not fit to how subjects where stored [here](https://github.com/corona-school/backend/pull/107/files#diff-3abc81a34a2dbec883e655069ec042c0R150) and displayed [there](https://github.com/corona-school/backend/pull/107/files#diff-3abc81a34a2dbec883e655069ec042c0R217). We should either change the dev data or the way how subjects are stored and displayed.

Reply:
changed dev data [https://github.com/corona-school/backend/commit/442838bde60b2004c7362ce80a02293802544f9b](https://github.com/corona-school/backend/commit/442838bde60b2004c7362ce80a02293802544f9b)